css/style.css

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap');
+
+/* Default Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+/* HEADER STYLES */
+header {
+  width: 100vw;
+  height: 100vh;
+  padding: 20px 40px;
+  background-image: url(../images/header-shapes\ mobile-bg.jpg);
+  background-size: contain;
```

Comment:
- [ ] Just as the previous reviewer pointed out, kindly apply her well-explained response and fix the image positioning. 
I've attached a comparison of your current design and that of the Figma template for your self-assessment. good job so far 👏  

## Your Design:
![image](https://user-images.githubusercontent.com/50990925/212292145-f284ffe6-cf82-46e3-8710-489e65f2b363.png)

## Original Figma Design:
![image](https://user-images.githubusercontent.com/50990925/212292926-497cae16-b5ce-4c98-ae3d-adf0904aa03d.png)


## Figma Design as Adjusted by previous reviewer with your design:
![image](https://user-images.githubusercontent.com/50990925/212292642-26984e20-5c00-4143-8071-00fd83cab60e.png)